app/src/main/java/com/worldsnas/starplayer/model/LocalMusicProvider.kt

```diff
@@ -2,5 +2,5 @@ package com.worldsnas.starplayer.model
 
 interface LocalMusicProvider {
 
-    suspend fun getAllMusic() : List<Music>
+    suspend fun getAllMusic() : List<LocalMusic>
```

Comment:
in dorost nist
ma object aslimon hamon `Music` hastesh va local music estefade dige i dare

Reply:
hame tabdil ha ro dakhel Repository anjam bedam, doroste?

Reply:
hefze object `Music` kheili moheme
va bale tabdil ha bayad onja anjam beshe
baraye sample mitonid repo haye [in project mano](https://github.com/worldsnas/aio) ro bebinid